Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -783,6 +821,8 @@ public extension Purchases {
 #endif
 
     @objc func invalidateCustomerInfoCache() {
+        guard !self.systemInfo.dangerousSettings.minimalImplementationOnly else { return }
```

Comment:
torn on this one, it might be useful

Reply:
Yeah this type of stuff is scary. We might need to have a lot of _duplicate_ test coverage with this flag on so we can verify its behavior.